packages/admin-ui/ui/src/domain/orders/new/components/shipping-details.tsx

```diff
@@ -1,13 +1,13 @@
 import qs from "query-string"
```

Comment:
question: is this meant to be included?

Reply:
We use it on line 35, we need to install it as a dependency. It was working in dev because `query-string` exists in the node_modules of the monorepo, but when we ship everything it is no longer available. 

I think we should fix this component so it doesn't rely on doing a direct GET request (and `query-string` to format the params), and instead use `medusa-react`, but should be a separate PR.